Tests/LegacyUnit/Service/RegistrationManagerTest.php

```diff
@@ -6386,11 +6385,13 @@ public function createRegistrationIncreasesRegistrationCountInEventFromZeroToOne
         );
 
         $subject->createRegistration($this->seminar, [], $plugin);
-
-        $seminarData = \Tx_Oelib_Db::selectSingle('*', 'tx_seminars_seminars', 'uid = ' . $this->seminarUid);
+        $connection = $this->getConnectionForTable('tx_seminars_seminars');
+        $seminarData = $connection->select(['*'], 'tx_seminars_seminars', ['uid' => $this->seminarUid])->fetch();
 
         $registrationUid = $subject->getRegistration()->getUid();
-        \Tx_Oelib_Db::delete('tx_seminars_attendances', 'uid = ' . $registrationUid);
+
+        $connection = $this->getConnectionForTable('tx_seminars_attendances');
```

Comment:
Please let's use separate variables for the two connections (instead of reusing one variable), e.g., `$seminarsConnection` and `$attendancesConnection`.